linux-tkg-config/prepare

```diff
@@ -338,6 +343,10 @@ _tkg_srcprep() {
 
   msg2 "Setting version..."
   scripts/setlocalversion --save-scmversion
+  if [ $_basever -ge 512 ] && [[ "$lto_enable" =~ [yY] ]] || [ -n "$_lto_mode" ]; then
+    scripts/config -e LTO_CLANG_THIN 
```

Comment:
Better be safe here and explicitly disable full lto with `scripts/config -d LTO_CLANG_FULL`, since a user may have a full LTO config from the currently running kernel and wants to switch to thin LTO.